sdk/compute/Azure.ResourceManager.Compute/api/Azure.ResourceManager.Compute.netstandard2.0.cs

```diff
@@ -5727,6 +5808,7 @@ public partial class VirtualMachineScaleSetOSProfile
         public string ComputerNamePrefix { get { throw null; } set { } }
         public string CustomData { get { throw null; } set { } }
         public Azure.ResourceManager.Compute.Models.LinuxConfiguration LinuxConfiguration { get { throw null; } set { } }
+        public bool? RequireGuestProvisionSignal { get { throw null; } set { } }
```

Comment:
Could it be `IsGuestProvisionSignalRequired`?

Reply:
after i renamed it, i am getting build error:
 Member 'public System.Nullable<System.Boolean> Azure.ResourceManager.Compute.Models.VirtualMachineOSProfile.RequireGuestProvisionSignal.get()' does not exist in the implementation but it does exist in the contract.
 how can i resolve this?

Reply:
@ArcturusZhang

Reply:
this property already existed so it will remain unchanged: https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/compute/Azure.ResourceManager.Compute/src/Generated/Models/VirtualMachineOSProfile.cs#L31